src/components/Navbar/Searchbar.js

```diff
@@ -1,14 +1,13 @@
-import React from 'react';
-
-import styles from './Searchbar.module.scss';
+import React from 'react'
```

Comment:
Remember to add a space between imports. (Order: internal libraries, internal utils, internal components, assets, styles)